<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JIMMessenger</class>
 <widget class="QMainWindow" name="JIMMessenger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JIMMessenger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textChatEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushAdd">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDelete">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>20</width>
      <height>411</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSend">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushCancle">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>320</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listContacts">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>271</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="chatDisplay">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="chatName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="myImg">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>360</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="contactImg">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>50</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="myName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLogin"/>
    <addaction name="actionLogoff"/>
    <addaction name="separator"/>
    <addaction name="actionRegister"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionUlined"/>
    <addaction name="actionItalic"/>
    <addaction name="actionBold"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionItalic"/>
   <addaction name="actionBold"/>
   <addaction name="actionUlined"/>
   <addaction name="separator"/>
   <addaction name="actionsmile"/>
   <addaction name="actioncrazy"/>
   <addaction name="actionsad"/>
  </widget>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionLogoff">
   <property name="text">
    <string>Logoff</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="text">
    <string>Register</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset>
     <normalon>i.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset>
     <normalon>b.jpg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionUlined">
   <property name="icon">
    <iconset>
     <normaloff>u.jpg</normaloff>
     <normalon>u.jpg</normalon>u.jpg</iconset>
   </property>
   <property name="text">
    <string>Ulined</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionsmile">
   <property name="icon">
    <iconset>
     <normaloff>ab.gif</normaloff>ab.gif</iconset>
   </property>
   <property name="text">
    <string>smile</string>
   </property>
  </action>
  <action name="actioncrazy">
   <property name="icon">
    <iconset>
     <normaloff>ai.gif</normaloff>ai.gif</iconset>
   </property>
   <property name="text">
    <string>crazy</string>
   </property>
  </action>
  <action name="actionsad">
   <property name="icon">
    <iconset>
     <normaloff>ac.gif</normaloff>ac.gif</iconset>
   </property>
   <property name="text">
    <string>sad</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
